addons/website_sale/views/res_config_settings_views.xml

```diff
@@ -172,6 +172,25 @@
                             </div>
                         </div>
                     </setting>
+                    <setting
+                        id="shipping_provider_shiprocket_setting"
```

Comment:
And the module is auto install do you really need to add an extra setting for it since the setting for the shipping method already exists?

Reply:
This setting is for the Shiprocket provider not for the Cash on Delivery :) same as the UPS provider.
Thanks!

Reply:
Okay got it, it was missing in the ecommerce settings